composer.json

```diff
@@ -19,7 +19,7 @@
   "require": {
     "php": ">=5.5",
     "symfony/symfony": "^2.6",
-    "jms/serializer": "^1.0"
+    "jms/serializer-bundle": "^0.13|^1.0"
```

Comment:
It's using a service definition provided by the bundle:
https://github.com/treehouselabs/base-api-bundle/blob/master/src/TreeHouse/BaseApiBundle/Controller/BaseApiController.php#L237